src/ResourceManager/Websites/Commands.Websites/Cmdlets/BackupRestore/RestoreAzureDeletedWebApp.cs

```diff
@@ -0,0 +1,179 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.WebApps.Cmdlets.WebApps;
+using Microsoft.Azure.Commands.WebApps.Models;
+using Microsoft.Azure.Management.WebSites.Models;
+using System;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.WebApps.Cmdlets.BackupRestore
+{
+    /// <summary>
+    /// Restores a deleted Azure Web App's contents and settings to an existing web app
+    /// </summary>
+    [Cmdlet("Restore", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "DeletedWebApp"), OutputType(typeof(PSSite))]
```

Comment:
This should have a DefaultParameterSetName attribute

Reply:
Please also add help files for your change: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md

Reply:
Also please update the changelog at src/Websites/Commands.Websites/Changelog.md